getting_started/step_by_step/your_first_game.rst

```diff
@@ -241,23 +240,23 @@ and use ``clamp()`` to prevent it from leaving the screen:
         position.y = clamp(position.y, 0, screensize.y)
 
 
-.. tip:: *Clamping* a value means restricting it to a given minimum/maximum range.
+.. tip:: *Clamping* a value means restricting it to a given range.
 
-Click "Play the Edited Scene. (F6)" and confirm you can move the player
+Click "Play the edited scene" (``F6``) and confirm you can move the player
 around the screen in all directions.
```

Comment:
I've noticed the run shortcuts are changed now. The button labels are "Play Scene (Command+R)" and "Play (Command+B)".

Reply:
It's changed only for macOS though.

Reply:
Shortcuts are different, but the label text is the same, right? It used to say "Play the Edited Scene", and now it says "Play Scene" plus "Play the edited scene."

Reply:
Yes, the tooltip is the same for every platform, but the shortcuts are not. So either put both shortcuts or none, otherwise I think it may cause confusion.